src/js/components/molecules/wizard-step-download-list/wizard-step-download-list.vue.html

```diff
@@ -10,7 +11,7 @@
                           :setSelectedItem="setSelectedItem" />
         </li>
     </ul>
-    <button @click="handleSubmit" type="submit" class="wizard-step-download-list__button button" v-if="submitButton">
+    <button class="wizard-step-download-list__button button" v-if="submitButton">
```

Comment:
Should probably be disabled by default, and depending on OS detection become enabled?

Reply:
Windows is default selected :)